backend/pubsub.js

```diff
@@ -20,13 +20,13 @@ import type {
   NotificationTypeEnum, ResponseTypeEnum
 } from '~/shared/pubsub.js'
 
-import type { JSONType } from '~/shared/types.js'
+import type { JSONType, JSONObject } from '~/shared/types.js'
 
 const { bold } = require('chalk')
 const WebSocket = require('ws')
 
 const { PING, PONG, PUB, SUB, UNSUB } = NOTIFICATION_TYPE
-const { ERROR, SUCCESS, PUSH_ERROR } = RESPONSE_TYPE
+const { ERROR, SUCCESS } = RESPONSE_TYPE
```

Comment:
Just something to keep an eye on between your PR @SebinSong and @snowteamer's recent modifications to the pubsub in #1792 (e.g. `SUCCESS` => `OK`). If this PR is merged first @snowteamer will need to update this code as well.

Reply:
@taoeffect This PR doesn't contain any change that uses `REQUEST.SUCCESS`, so it's safe.